priv/repo/data_migrations/migrate_add_position_order_to_ideas.exs

```diff
@@ -0,0 +1,39 @@
+defmodule Mindwendel.Repo.DataMigrations.MigrateAddPositionOrderToIdeas do
+  require Logger
+  alias Mindwendel.Repo
+  alias Mindwendel.Brainstormings.Idea
+  alias Mindwendel.Brainstormings.Brainstorming
+  import Ecto.Query, warn: false
+
+  def run do
+    idea_query = from(idea in Idea,
+                   where: is_nil(idea.position_order),
+                   distinct: idea.brainstorming_id
```

Comment:
This may be fine as this is supposed to only be run once, but generally speaking migrations shouldn't rely on application code. Application code changes over time, which means the migrations may break or worse change behavior. 

Ecto can be used without schemas which is great for this kind of use case: https://hexdocs.pm/ecto/schemaless-queries.html